internal/util/testutil/equal_test.go

```diff
@@ -41,7 +41,6 @@ func TestEqual(t *testing.T) {
 	AssertNotEqual(t, math.Inf(-1), math.Inf(+1))
 
 	AssertEqual(t, 0.0, math.Copysign(0, +1))
-	AssertNotEqual(t, math.Copysign(0, +1), math.Copysign(0, -1))
```

Comment:
I assume it was here because we used to support -0. I want to remove this because when compat returns -0 and target returns 0, we want to say they are equal. Should this cause issue? If this change is undesirable, we can test this solely in diff and undo this change.

Reply:
Well, even if we don't support -0 anymore, we probably want to make sure that if by mistake we return `-0` and compare it with `0`, the result of the comparison will tell us that they are not the same. With that, keeping this line of code looks reasonable to me.

Reply:
Sure let's test this edge case in diff test and undo this change